src/livox_points_plugin.cpp

```diff
@@ -399,30 +397,28 @@ void LivoxPointsPlugin::PublishPointCloud2XYZI(std::vector<std::pair<int, AviaRo
 
     pcl::PointCloud<pcl::PointXYZI> pc;
     pc.points.resize(points_pair.size());
-    ros::Time timestamp = ros::Time::now();
+    const auto timestamp = ros::Time::now();
     int pt_count = 0;
 #pragma omp parallel for
     for (int i = 0; i < points_pair.size(); ++i) {
```

Comment:
I think that this could be re-written as:
```suggestion
    ffor (const auto& [index, rotate_info] : points_pair) { // std::pair<int, gazebo::AviaRotateInfo>
```

Reply:
https://en.cppreference.com/w/cpp/language/structured_binding

Reply:
I always wanted to use this feature :raised_hands: